index.html

```diff
@@ -274,6 +274,44 @@ <h2 class="list-title">Skills</h2>
             </li>
         </ul>
     </section>
+
+    <footer id="about-section">
+        <div class="form-container">
+            <p class="form-text">
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </p>
+
+            <form class="form-container" action="https://formspree.io/f/xdoyzdjj" method="POST">
+                <input class="form-name" type="text" name="name" placeholder="Full name" maxlength="30" required />
+                <input class="form-email" type="email" name="email" placeholder="Email address" required />
+                <textarea class="form-text-area" name="message" cols="30" rows="10" maxlength="500">
```

Comment:
- [ ] Add a required attribute here

Reply:
Thank you for the review. The 'required' attribute has been added just now.

Reply:
Well addressed this!